changes/1036.feature.md

```diff
@@ -0,0 +1 @@
+Enable small letter string for query filtering item.
```

Comment:
Note: enum values are usually implementation details, i.e., hidden under the abstraction. I think it would be better to stick on the names only with relaxation of the case sensitivity when matching them.

Reply:
I agree with you. I updated the code to enable only enum names, with small letter string.